lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_method.rb

```diff
@@ -218,10 +233,14 @@ def self.run_method(cmd)
           yield stdin if block_given?
           stdin.close
           threads << Thread.new do
-            stdout.each_line { |msg| $miq_ae_logger.info("Method STDOUT: #{msg.strip}") }
+            stdout.each_line do |msg|
+              $request_log.request_info("Method STDOUT: #{msg.strip}", {:resource_id => miq_request_id}, false)
```

Comment:
Hmm - this didn't come out the way I'd expect and I think I led you down the wrong path.

I _think_ maybe instead of RequestLogger, we should have modified the AutomateLogger to accept a request_id.  Then the AutomateLogger would additionally write to the Request table (or a simpler RequestLogger).